src/Scratch.as

```diff
@@ -466,6 +468,7 @@ public class Scratch extends Sprite {
 		removeLoadProgressBox();
 		System.gc();
 		if (autostart) runtime.startGreenFlags(true);
+		loadInProgress = false;
```

Comment:
Are you confident that it's safe to set this to `false` before `stepRuntime()` calls `runtime.installNewProject()`?